flang/docs/FlangDriver.md

```diff
@@ -220,6 +220,13 @@ static or shared library, the generated library does not need a `main`
 function, as a final link stage will occur that will provide the `Fortran_main`
 library when creating the final executable.
 
+On Darwin, the logical root where the system libraries are located (sysroot)
+must be specified. This can be done with the cmake build flag `DEFAULT_SYSROOT`
+or by using the `-isysroot` flag when linking a binary. On other targets
+`-isysroot` doesn't change the linker command line. While with Clang
+`-isysroot` also changes the sysroot for includes, with Flang it only affects
+Darwin libraries' sysroot.
```

Comment:
This is worth expanding. Feel free to re-use.

```suggestion
On Darwin, the logical root where the system libraries are located (sysroot)
must be specified. This can be done with the CMake build flag `DEFAULT_SYSROOT`
or by using the `-isysroot` flag when linking a binary. On other targets
`-isysroot` doesn't change the linker command line (it only affects the header search path). While with Clang
`-isysroot` also changes the sysroot for includes, with Flang (and Fortran in general)
it only affects Darwin libraries' sysroot.
```

Reply:
Done. I've only changed the formatting, to fit everything within 80 columns.